presto-main/src/main/java/com/facebook/presto/execution/buffer/PageCodecMarker.java

```diff
@@ -13,36 +13,42 @@
  */
 package com.facebook.presto.execution.buffer;
 
+import com.google.common.annotations.VisibleForTesting;
+
 import java.util.Arrays;
 import java.util.stream.Collectors;
 
 import static com.google.common.base.Preconditions.checkArgument;
 
+/**
```

Comment:
I'll remove the static helpers that are `@VisibleForTesting`. I'll also remove the `.of` method and rename `.clear` to `.unset`. Tests will be slimmed, but I'm going to keep the style of looping over `PageCodec.values()` in the tests instead of hard coding `COMPRESSED` and `ENCRYPTED`. The looping over all possible values strategy prevents someone from adding a new marker type that accidentally uses a conflicting mask with other markers.